.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,17 @@
+name: Feature request
+description: Use this template for requesting new features. Please search existing issues first.
+labels: feature, needs-triage
```

Comment:
Nit: I'd remove `feature`, since we planned to remove the `enhancement`, `bug`, etc. in favor of using issue types